src/test/behave_tests/features/ceph_osd_test.feature

```diff
@@ -0,0 +1,129 @@
+@osd
+Feature: Install a basic Ceph cluster
+  In order to be able to provide storage services
+  As an system administrator
+  I want to install a Ceph cluster in the following server infrastructure:
+    - 3 nodes with 8Gb RAM, 4 CPUs, and 3 storage devices of 20Gb each.
+    - Using Fedora32 image in each node
+    - Configure ceph cluster in following way
+      - with number of OSD 0
+
+
+  Scenario: Create OSDs
+    Given I log as root into ceph-node-00
+    When I execute in cephadm_shell
+        """
+        ceph orch device ls
+        """
+    Then I wait for 60 seconds until I get
+        """
+        ceph-node-00.cephlab.com  /dev/vdb  hdd  Unknown  N/A    N/A    Yes
+        ceph-node-01.cephlab.com  /dev/vdb  hdd  Unknown  N/A    N/A    Yes
+        ceph-node-02.cephlab.com  /dev/vdb  hdd  Unknown  N/A    N/A    Yes
+        """
+    Then I execute in cephadm_shell
+        """
+        ceph orch daemon add osd ceph-node-00.cephlab.com:/dev/vdb
+        ceph orch daemon add osd ceph-node-01.cephlab.com:/dev/vdb
+        ceph orch daemon add osd ceph-node-02.cephlab.com:/dev/vdb
+        """
+    Then I execute in cephadm_shell
+        """
+        ceph orch device ls
+        """
+    Then I wait for 60 seconds until I get
+        """
+        ceph-node-00.cephlab.com  /dev/vdb  hdd  Unknown  N/A    N/A    No
+        ceph-node-01.cephlab.com  /dev/vdb  hdd  Unknown  N/A    N/A    No
+        ceph-node-02.cephlab.com  /dev/vdb  hdd  Unknown  N/A    N/A    No
+        """
+    Then I execute in cephadm_shell
+        """
+        ceph -s
+        """
+    Then I get results which contain
+        """
+        services:
+          mon: 3 daemons, quorum ceph-node-00.cephlab.com,ceph-node-01,ceph-node-02
+          osd: 3 osds: 3 up
+        """
+
+
+  Scenario: Create block devices
```

Comment:
Is better to separate the RBD test functionality from the OSD test functionality. 
Several reasons:
1. Separate functionality makes easy to test different things.
2. Errors and problems are more easy to isolate and debug
3. We do not need to repeat all the steps and checks needed to reach to have a basic test cluster  and test a more complex functionality like RBD. For exampl in the RBD test you can start with a ceph cluster like the one used for "cephadm commands" , and avoid all the previous scenario.

so we need to have one test for testing OSD creation
and anohther different one for RBD test